src/services/videomania/index.jsx

```
@@ -0,0 +1,208 @@
+import ErrorMessage from "../../components/form/ErrorMessage";
+import Form from "../../components/form/Form";
+import { Spinner } from "../../components/theme/Loader";
+import AppTextArea from "../../components/form/AppTextArea";
+
+import { getVideoSubCategoryByCategory } from "../../app/features/videomania";
+import { FaUpload } from "react-icons/fa";
+import videoIcon from "../../assets/videoIcon.svg";
+import { Toast } from "../../components/theme/Toast";
+
+export const AddVideoMania = ({
+  setAddModal,
```

ESLint

[error] 'setAddModal' is missing in props validation

(react/prop-types)